src/main/java/net/dv8tion/jda/core/entities/Guild.java

```diff
@@ -886,6 +886,99 @@ default Emote getEmoteById(long id)
     @CheckReturnValue
     RestAction<List<Ban>> getBanList();
 
+    /**
+     * Retrieves a {@link net.dv8tion.jda.core.entities.Guild.Ban Ban} of the provided ID
+     * <br>If you wish to ban or unban a user, either please use {@link GuildController#ban(User, int) GuildController.ban(User, int)} or
```

Comment:
`please use either` not `either please use`

Reply:
Also considering this is for `id` I would refer to `ban(id, int)` and `unban(id)`.

Reply:
Same applies for the unban right?